tests/vm-prep

```diff
@@ -64,6 +64,12 @@ prepare()
 		$VIRSH net-undefine $NETWORK_NAME
 	fi
 
+    # In case a network has leaked in from elsewhere
+    if ip address show dev $NETWORK_NAME >/dev/null 2>/dev/null; then
+        ip link set $NETWORK_NAME down || true
+        brctl delbr $NETWORK_NAME || true
+    fi
+
```

Comment:
Right now the indentation looks really broken and confusing (like this new block being outside of the function); can it be consistently changed to 4 spaces instead of a tab then?

Thanks for the explanation about the bridge handling -- and "d'oh!" ☺